users/admin_user_proxy.py

```diff
@@ -13,6 +14,16 @@ def sms_conversations_field(self, obj):
     )
 
 
+@admin_field(short_description="Impersonation", allow_tags=True)
+def impersonate_field(self, obj):
+    if obj.pk:
+        return make_button_link(
+            reverse("admin:impersonate-user", kwargs={"user_id": obj.pk}),
+            f"Impersonate {obj.full_name}\u2026",
```

Comment:
This ellipsis here made it look like the text was cut off when it wasn't. I'm wondering if an arrow may be more appropriate? Or no especial character at all?

Reply:
Oh heh, this is actually a UX convention we use throughout the admin parts of our site, whenever an action will open up a new page to confirm the action rather than actually execute the action immediately.  It's meant to echo the same use of [ellipses in pull-down menus](https://ux.stackexchange.com/questions/9544/what-is-the-significance-of-the-three-dots-on-menus-and-buttons-and-how-to).  But maybe it's inappropriate since this isn't a pull-down menu?

I _do_ think there should be some indication that clicking the button will give the user more information about the action, and allow them to confirm it, rather than immediately executing the action, whether that's done through an ellipsis or some other means.  We should revisit later, and if we decide to use something else then we should implement it across the board!